distribution/resources/conf/default-micro-gw.conf.template

```diff
@@ -191,6 +191,14 @@
 
 # Throttling configurations
 [throttlingConfig]
+  [throttlingConfig.nodeLocal]
```

Comment:
Can we add more documentation comments here with which units (e.g.seconds ..) and etc.